packages/renderer/src/lib/container/ContainerColumnActions.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+import { ContainerUtils } from './container-utils';
+import ContainerColumnActionsCompose from './ContainerColumnActionsCompose.svelte';
+import ContainerColumnActionsContainer from './ContainerColumnActionsContainer.svelte';
+import ContainerColumnActionsPod from './ContainerColumnActionsPod.svelte';
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+
+const containerUtils = new ContainerUtils();
+</script>
+
+{#if containerUtils.isContainerGroupInfoUI(object)}
+  {#if object.type === ContainerGroupInfoTypeUI.POD}
+    <ContainerColumnActionsPod object="{object}" />
+  {/if}
+  {#if object.type === ContainerGroupInfoTypeUI.COMPOSE}
```

Comment:
```suggestion
  {:else if object.type === ContainerGroupInfoTypeUI.COMPOSE}
```

An object type cannot be POD and COMPOSE at the same time

Reply:
Fixed.